libs/common/src/lib/user/onboarding/onboarding.component.ts

```diff
@@ -79,6 +79,9 @@ export class OnboardingComponent {
       if (enabled) {
         this.onboardingForm.get('phoneInternationalCode')?.setValidators([Validators.required]);
```

Comment:
According to the docs,`addValidators`could be used in case the number format validator should be preserved